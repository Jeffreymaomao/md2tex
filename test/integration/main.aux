\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\@writefile{toc}{\contentsline {section}{\numberline {1}標題 1}{1}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}標題 2}{1}{subsection.1.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsubsection}{\numberline {1.1.1}標題 3}{1}{subsubsection.1.1.1}\protected@file@percent }
\@writefile{lol}{\contentsline {lstlisting}{/}{2}{lstlisting.-1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {2}指定語言的代碼塊}{2}{section.2}\protected@file@percent }
\@writefile{lol}{\contentsline {lstlisting}{p}{2}{lstlisting.-3}\protected@file@percent }
\gdef \@abspage@last{2}
