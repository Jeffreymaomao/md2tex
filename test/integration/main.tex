
% Sets the document class and font size
\documentclass[12pt]{article}
\usepackage[a4paper, margin=1in]{geometry}

\usepackage[utf8]{inputenc}    % Input encoding
\usepackage[T1]{fontenc}       % Font encoding
\usepackage{fontspec}          % font encoding

% Advanced math typesetting
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{mathtools}
\usepackage{physics}

% Symbols and Text
\usepackage{bbold}     % bold font
\usepackage{ulem}      % strikethrough
\usepackage{listings}  % Source code listing
\usepackage{import}    % Importing code and other documents

% graphics
\usepackage[dvipsnames]{xcolor}
\usepackage{graphicx}
\usepackage{tikz}
\usepackage{pgfplots}
\usepackage{tcolorbox}

% figure
\usepackage{float}
\usepackage{subfigure}

\usepackage{hyperref}  % Hyperlinks in the document
\hypersetup{
    colorlinks=true,
    linkcolor=Blue,
    filecolor=red,
    urlcolor=Blue,
    citecolor=blue,
    pdftitle={Article},
    pdfauthor={Author},
}

\usepackage{xeCJK}             % Chinese, Japanese, and Korean characters
\setCJKfamilyfont{kai}{標楷體}
%================================================================================================
\begin{document}

\section{標題 1}

\subsection{標題 2}

\subsubsection{標題 3}

\textbf{標題 4}

\textbf{標題 5}

\textbf{標題 6}

\begin{quote}
	這是一個引用。
	\begin{quote}
	嵌套引用。
	\end{quote}


\end{quote}

\begin{quote}
	這是另一段引用
\end{quote}


\begin{itemize}
	\item A 無序列表項 1
	\item B 無序列表項 2
	\begin{itemize}
		\item C 無序子列表項 1
		\item D 無序子列表項 2

	\end{itemize}
	\item E 無序列表項 1
	\item F 無序列表項 2
	\begin{enumerate}
		\item G 有序子列表項 1
		\item H 有序子列表項 2

	\end{enumerate}
	\item I 無序列表項 1
	\item J 無序列表項 2
	\begin{itemize}
		\item K 有序子列表項 1
		\begin{itemize}
			\item L 有序子列表項 2
			\item M 有序子列表項 1

		\end{itemize}
		\item N 有序子列表項 2

	\end{itemize}

\end{itemize}
\begin{enumerate}
	\item O 有序列表項 1
	\item P 有序列表項 2
	\begin{enumerate}
		\item Q 有序子列表項 1
		\item R 有序子列表項 2

	\end{enumerate}
	\item S 有序列表項 1
	\begin{enumerate}
		\item T 有序子列表項 2

	\end{enumerate}

\end{enumerate}

\begin{equation}
y = ax+b
\end{equation}


\lstinline{行內代碼} 示例。

\vspace{5pt}
\hrule
\vspace{6pt}


\begin{equation}
y = ax+b
\end{equation}


\begin{lstlisting}% [ language = javascript ]
// 代碼塊
console.log('Hello, Markdown!');
\end{lstlisting}

\href{http://example.com}{鏈接文本}

| 表格頭 | 表格頭 | 表格頭 || ------ | ------ | ------ || 單元格 | 單元格 | 單元格 || 單元格 | 單元格 | 單元格 |

\vspace{5pt}
\hrule
\vspace{6pt}

\textit{斜體文本} 或 \textit{斜體文本}

\textbf{粗體文本} 或 \textbf{粗體文本}

\textbf{\textit{粗斜體文本}} 或 \textbf{\textit{粗斜體文本}}

\sout{刪除線文本}

\begin{itemize}
	\item [ ] 任務列表未完成
	\item [x] 任務列表已完成

\end{itemize}
\lstinline{行內代碼}


\begin{lstlisting}
沒有指定語言的代碼塊
\end{lstlisting}

\section{指定語言的代碼塊}

\begin{lstlisting}% [ language = python ]
print("Hello, Markdown!")
\end{lstlisting}

$\text{如果整行都是inline-math}$

單行數學公式
\begin{equation}
y = ax^2 + bx + c
\end{equation}


多行數學公式
\begin{equation}
\begin{aligned}
x = \cos(t)\\
y = \sin(t)
\end{aligned}
\end{equation}



這是一個自動鏈接：\href{http://google.com}{google}

數學公式 $\sqrt{x}$ 或是 $\frac{1}{2}$

普通段落文本。支持\textit{斜體}、\textbf{粗體}、\sout{刪除線}。




%================================================================================================
\end{document}
