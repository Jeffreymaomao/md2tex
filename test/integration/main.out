\BOOKMARK [1][-]{section.1}{\376\377\152\031\230\114\000\040\0001}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{\376\377\152\031\230\114\000\040\0002}{section.1}% 2
\BOOKMARK [3][-]{subsubsection.1.1.1}{\376\377\152\031\230\114\000\040\0003}{subsection.1.1}% 3
\BOOKMARK [1][-]{section.2}{\376\377\143\007\133\232\212\236\212\000\166\204\116\343\170\274\130\112}{}% 4
